### PR TITLE
Adding Pipewire watch recipe

### PR DESCRIPTION
**Pull request by dixiedream**
A simple volume widget for pure Pipewire systems.

### CODE REVIEW

File: recipes/watch.mdwn [hunk 1]
```diff
@@ -146,6 +146,24 @@ local mpris, mpris_timer = awful.widget.watch(
 )
 ```
 
+## pipewire
+```lua
```
Comment (Aire-One) [inline recipes/watch.mdwn:150]: I think we need an extra empty line here to allow the markdown interpreter to work :

```suggestion
## pipewire

```lua
```

**Review by Aire-One — CHANGES_REQUESTED** [review]

**Comment by dixiedream** [comment]
@Aire-One Thanks!

**Review by Aire-One — APPROVED** [review]
Thank you!

**Review by Elv13 — APPROVED** [review]

**Comment by psychon** [comment]
Correct me if I am wrong, but `pactl` is part of PulseAudio and not pipewire.
```
$ man -k pactl
pactl (1)            - Control a running PulseAudio sound server
```

**Comment by dixiedream** [comment]
You're absolutely right but it will also been installed by `pipewire-pulse` which is the only package you need in order to make pulse based apps on pipewire.

**Pull request merged**